### PR TITLE
Feature/web layout

### PR DESCRIPTION
**Pull request by DemogorGod**
Morning fellas!

This is just an update on our web page made the layout and side nav bar so that way I can work on connecting with the dummy wallets that Chris has set up. 

added some slick animation with a good router flow.

check it out!

https://user-images.githubusercontent.com/63758598/203294432-a64bba92-e33a-40f6-a44d-cf55d49b26c6.mov

### CODE REVIEW

File: apps/web/tsconfig.json [hunk 1]
```diff
@@ -21,5 +21,4 @@
       "./node_modules/@types"
     ]
   },
-  "include": ["./src/**/*.ts", "./src/**/*.vue"]
-}
+  "include": ["./src/**/*.ts", "./src/**/*.vue", "src/main.ts", "src/composables/router.ts", "src/composables/router.ts", "src/main.ts"]}
```
Comment (shanejearley) [inline apps/web/tsconfig.json:24]: Change to `"include": ["./src/**/*.ts", "./src/**/*.vue"]`.

**Review by shanejearley — CHANGES_REQUESTED** [review]
Looks nice!

File: apps/web/tsconfig.json (hunk 1 above)
Comment (DemogorGod) [reply apps/web/tsconfig.json]: done

**Review by shanejearley — APPROVED** [review]

**Comment by ccali11** [comment]
> Looks nice!

Agreed!

**Pull request merged**